test/decryption_test.rb

```diff
@@ -34,7 +34,7 @@ def test_it_can_decrypt_a_message
     @shifts.stubs(:random_number).returns(2715)
     Date.stubs(:today).returns(Date.new(1995, 8, 04))
 
-    assert_equal "hello world", @decryption.decrypt_message("keder ohulw", @shifts.shifts.values)
+    assert_equal "hello world", @decryption.decrypt_message("keder ohulw", @shifts.shifts)
```

Comment:
Removed .values call from @shifts.shifts due to the change in logic of the shifts argument to return array